lib/content/utils.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const _ = require('lodash');
+const isUUID = require('validator/lib/isUUID');
+
+
+const check = {
+
+  /**
```

Comment:
@Snugug sentence phrasing confusing. So, for `bundled together in to check in the module.exports`, I think you mean that there will be separate functions, but they would be called like this:

```
module.exports = {
  check: {
    id,
    revision,
    type,
  },
  other,
  functions,
}
```

correct?